benchmark/benchmain/main.go

```diff
@@ -255,6 +261,9 @@ func unconstrainedStreamBenchmark(start startFunc, stop ucStopFunc, bf stats.Fea
 				}
 				recver(pos)
 				atomic.AddUint64(&resp, 1)
+				if maxSleep > 0 {
+					time.Sleep(time.Duration(math_rand.Intn(maxSleep)))
+				}
```

Comment:
Is this client code or server code, or is this used for both sides?

I think we might want the sender in both directions to be sleeping, and not sleeping after receiving.  If the sender isn't also sleeping, it could fill up buffers and have more impact than you intended.

Reply:
Yes, you are right. I'll remove the delay from the client receive goroutine and instead will add it [here](https://github.com/grpc/grpc-go/blob/098b2d00c5bcbc5c696dcab9a2a7f7b442f8142e/benchmark/benchmark.go#L137-L145)

In order to do that I'll need to pass the sleepBetweenRPCs parameter to the server (probably I'll add ti to SimpleRequest rpc)

Reply:
Fixed. I end up using metadata to pass the sleep duration to the server. [SimpleRequest](https://github.com/grpc/grpc-proto/blob/master/grpc/testing/messages.proto#L69) protobuf is used in too many places across languages, so I am not sure if adding fields to it is a good idea.